src/Header/SecondaryMenu.tsx

```diff
@@ -0,0 +1,39 @@
+import classnames from 'classnames';
+import React from 'react';
+import {Social} from '../Social';
+
+export interface SecondaryMenuProps {
+  className?: string;
+}
+
+export const SecondaryMenu: React.SFC<SecondaryMenuProps> = ({
+  className,
+  ...attributes
+}) => {
+  return (
+    <div {...attributes} className={classnames('secondary-menu', className)}>
+      <div className="secondary-menu__item secondary-menu__item_lykke-streams hidden">
```

Comment:
Not sure why all menu items are hidden

Reply:
Pages related to these menu items aren't ready yet, so we will unhide them later